### ISSUE TITLE
Students: Read this!

### ISSUE THREAD

**Issue by terriko** [python-gsoc/gsoc#87]
Please read  [How to get started with GSoC](https://python-gsoc.org/index.html#gettingstarted) first!

This repo isn't a great place to look for a first commit during GSoC. (This website isn't a valid GSoC project because there isn't enough code or changes to employ a student for a few months.)  You probably want to look at the issue trackers for our sub-orgs.  [You can see the list of python sub-orgs here](https://python-gsoc.org/index.html#ideas).  Have fun!

**Comment by veb7vmehra** [comment]
@terriko Is python-gsoc going to participate in GSOC this year? Also through which repository can I contribute for python-gsoc? Please guide

**Comment by terriko** [comment]
Yes, we are hoping to participate in GSoC 2020 (though, of course, it's up to Google if we get chosen.  We won't know for sure if we're participating until they make their choices on Feb 20, 2020.)  

All the info for python-gsoc is at the links I gave above, or you can start at the top of the https://python-gsoc.org/ website if you prefer.  Good luck!
